src/js-display-panel.ts

```diff
@@ -0,0 +1,135 @@
+import { ToggleStatus } from './js-feature-toggle-manager';
+import { QueryStringReader } from './github/gtmsportswear/js-utilities@1.0.0/js-utilities';
+
+export abstract class DisplayPanel {
+  public draw(): Element {
+    const panel = document.createElement('div');
+    panel.classList.add('display-panel');
+
+    return panel;
+  }
+}
+
+export class FeatureToggleDisplayPanel extends DisplayPanel {
+  constructor(private toggles: ToggleStatus[]) {
+    super();
```

Comment:
@StentorianCat in TypeScript, you have to do this. It's just something to do with the way the code is transpiled. If a class extends a parent class and has a constructor, it must call `super()` in the constructor.

Here's the documentation I'm referring to:
http://www.typescriptlang.org/docs/handbook/classes.html